gpii/node_modules/preferencesServer/src/preferencesService.js

```diff
@@ -83,7 +153,60 @@ var fluid = fluid || require("infusion");
         noUpdateOnSnapset: "Cannot update:  GPII key \"%gpiiKey\" is a snapset"
     });
 
-    // APIs for Preferences and Preferences Safes
+    /**
+     * Handler for creating cloud safe credentials.
+     *
+     * This handler will first fetch the preferences safe to make sure it
+     * exists and it active.  It will then take the username and password and
+     * create a new gpii-express-user entry.  With that we will then create
+     * the cloudsafe credential document that will contain the ID's of both
+     * the prefssafe and the user record.
+     *
+     * @param {Object} that - Instance of `gpii.preferencesServer.preferencesService`.
+     * @param {Object} options - Data used to create the new credentials.
+     * @param {Object} options.prefsSafeId - Id for the preferences safe we are adding
```

Comment:
Parameters stated at line 167-169 are strings instead of objects.

Reply:
Fixed.